app/renderer/components/main/main.js

```diff
@@ -402,12 +384,11 @@ class Main extends ImmutableComponent {
       windowActions.setDownloadsToolbarVisible(false)
     })
 
-    const self = this
     ipc.on(messages.BLOCKED_RESOURCE, (e, blockType, details) => {
-      const frameProps = frameStateUtil.getFrameByTabId(self.props.windowState, details.tabId)
-      frameProps && windowActions.setBlockedBy(frameProps, blockType, details.url)
+      windowActions.setBlockedBy(this.props.activeFrameKey, blockType, details.url)
     })
 
+    // TODO can we remove this?
     ipc.on(messages.BLOCKED_PAGE, (e, blockType, details) => {
```

Comment:
Not sure. The code that was commented out was added by @bridiver in a12cac496fc3dcbb59591c4e760ad0fde87476e0.